sda-commons-server-auth/src/test/java/org/sdase/commons/server/auth/key/OpenIdProviderDiscoveryKeySourceIT.java

```diff
@@ -142,7 +143,11 @@ void shouldLoadKeyWithTracing() {
               assertThat(spans)
                   .hasSize(2)
                   .extracting(
-                      s -> s.getAttributes().asMap().get(stringKey("http.url")), SpanData::getName)
+                      s ->
+                          s.getAttributes()
+                              .asMap()
+                              .get(stringKey(SemanticAttributes.URL_FULL.getKey())),
```

Comment:
What's the difference between `"http.url"` and `SemanticAttributes.URL_FULL.getKey()`?
Currently you are testing the same property that is used in production code. I would prefer to use a pure `String` (or a value defined in test scope) in tests. Using the same as in production will never fail - even if you (or the dependency) change something that breaks the contract - in this case maybe a dashboard.